base_layer/wallet/src/transaction_service/service.rs

```diff
@@ -1512,3 +1560,9 @@ where TBackend: TransactionBackend + Clone + 'static
     pub factories: CryptoFactories,
     pub config: TransactionServiceConfig,
 }
+
+#[derive(Clone, Copy)]
+enum PowerMode {
+    LowPower,
```

Comment:
nit: `Power` is redundant as context on variants is given by the enum type - `PowerMode::Low` and `PowerMode::Normal`